.github/workflows/proto.yml

```diff
@@ -3,28 +3,31 @@ name: Protobuf
 # This workflow is only run when a .proto file has been changed
 on:
   pull_request:
+    paths:
+      - 'proto/**'
+  push:
+    branches:
+      - master
```

Comment:
not sure if this needs to run on master?

Reply:
doesn't hurt